frontend/components/ClubPage/SocialIcons.tsx

```diff
@@ -71,7 +76,11 @@ const SocialIcons = ({ club }: SocialIconsProps): ReactElement => (
       .map((item) => (
         <SocialLine key={item.name}>
           <Icon style={iconStyles} name={item.icon} alt={item.icon} />{' '}
-          <SocialLink club={club} item={item} type={item.name} />
+          {item.name === 'address' ? (
+            <span>{club[item.name]}</span>
+          ) : (
+            <SocialLink club={club} item={item} type={item.name} />
+          )}
```

Comment:
It would be better to pass the address to `SocialLink` and then do the if statement there (similar to how email is doing it). At a future time, it might be nice to link the address to Google Maps.